docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Charmed Magma Orchestrator
+
+## Requirements
+
+The Access Gateway must be installed on a Kubernetes cluster with the following specifications:
+
+- **:material-kubernetes: Kubernetes**: A cluster with a minimum of 3 Nodes with 2 vCPUs and 8 GB of RAM each.
+
+## Deploy the magma-orc8r bundle
+
+From your Ubuntu machine, create an `overlay.yaml` file that contains the following content:
+
+```yaml
+applications:
+  orc8r-certifier:
+    options:
+      domain: <your domain name>
+  orc8r-nginx:
+    options:
+      domain: <your domain name>
+  tls-certificates-operator:
+    options:
+      generate-self-signed-certificates: true
+      ca-common-name: rootca.<your domain name>
+```
+
+
+!!! warning
+
+    This configuration is unsecure because it uses self-signed certificates.
```

Comment:
```suggestion
    This configuration is insecure because it uses self-signed certificates.
```

Reply:
I believe unsecure is correct, according to google/oxford: `(especially of a computer system) not protected against attack or other criminal activity.`

Reply:
Oxford dictionnary does not find it for me: https://www.oxfordlearnersdictionaries.com/us/spellcheck/english/?q=unsecure

It has `unsecured` however. Really curious where you find it, as I am unable to find it myself. This might be differences between US and UK English maybe?

Reply:
Keeping as is following our discussion

Reply:
Keeping as is following our discussion